modules/atomic-widgets/widgets/atomic-image.php

```diff
@@ -14,28 +14,33 @@
 }
 
 class Atomic_Image extends Atomic_Widget_Base {
-	public function get_icon() {
-		return 'eicon-image';
+	public function get_name() {
+		return 'a-image';
 	}
 
 	public function get_title() {
 		return esc_html__( 'Atomic Image', 'elementor' );
 	}
 
-	public function get_name() {
-		return 'a-image';
+	public function get_icon() {
+		return 'eicon-image';
 	}
 
 	protected function render() {
 		$settings = $this->get_atomic_settings();
 
-		$image_url = $settings['image'];
-
-		?> <img
-			src='<?php echo esc_url( $image_url ); ?>'
-			alt='Atomic Image'
-		/>
-		<?php
+		$attrs = array_filter( array_merge(
+			$settings['image'] ?? [],
+			[ 'class' => $settings['classes'] ?? '' ]
```

Comment:
what will happen here if you don't have an image transformer?

Reply:
Done